### DISCUSSION TITLE
How do I import AWS VPC S3 Endpoint?

### DISCUSSION THREAD

**Discussion by october-03** [GoogleCloudPlatform/terraformer#1647] [Q&A]
I'd like to import VPC S3 Endpoint that was previously created in the Console into Terraformer, is there any way? I tried importing VPC and S3, but it was failed.

**Comment by mmigliari** [comment]
As far as I can tell, there is no current support for aws_vpc_endpoint in terraformer.  You might want to submit a feature request.
